backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/search/predicate/impl/NestedPredicateBuilderImpl.java

```diff
@@ -23,24 +26,27 @@
 
 	private final String absoluteFieldPath;
 
+	private SearchPredicateContributor<Void, ? super ElasticsearchSearchPredicateCollector> nestedContributor;
+
 	NestedPredicateBuilderImpl(String absoluteFieldPath) {
 		this.absoluteFieldPath = absoluteFieldPath;
 	}
 
 	@Override
-	public ElasticsearchSearchPredicateCollector getNestedCollector() {
+	public Consumer<SearchPredicateContributor<Void, ? super ElasticsearchSearchPredicateCollector>> getNestedCollector() {
```

Comment:
Same here, you could remove this method and just promote `void nested(SearchPredicateContributor<Void, ? super ElasticsearchSearchPredicateCollector>)` to a public method.

Reply:
Done.